Documentation/ApiOverview/Database/ExpressionBuilder/_RepositoryConcat.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Repository;
+
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+
+final class MyTableRepository
+{
+    private const TABLE_NAME = 'pages';
+    public function __construct(private readonly ConnectionPool $connectionPool) {}
+
+    public function demonstrateConcat(): void
+    {
+        $queryBuilder = $this->connectionPool
+            ->getQueryBuilderForTable(self::TABLE_NAME);
+        $expressionBuilder = $queryBuilder->expr();
+        $result = $queryBuilder
+            ->select('uid', 'pid', 'title', 'page_title_info')
+            ->addSelectLiteral(
+                $expressionBuilder->concat(
+                    $queryBuilder->quoteIdentifier('title'),
+                    $queryBuilder->quote(' - ['),
+                    $queryBuilder->quoteIdentifier('uid'),
+                    $queryBuilder->quote('|'),
+                    $queryBuilder->quoteIdentifier('pid'),
+                    $queryBuilder->quote(']'),
+                ) . ' AS ' . $queryBuilder->quoteIdentifier('page_title_info'),
```

Comment:
Would using the new :php:`as` method here be the better example, showing best practice to utilize all new methods?

Reply:
Maybe, for now I took all examples from the changelog, we can do follow ups for more improvement later on

Reply:
No objection here for the documentation example. In the changelog I tried to not mix much with other new expression for explanations.